backend-lucene/src/main/java/org/hibernate/search/v6poc/backend/lucene/types/codec/impl/LuceneFieldFieldCodec.java

```diff
@@ -36,7 +38,15 @@ public void encode(LuceneDocumentBuilder documentBuilder, String absoluteFieldPa
 			return;
 		}
 
-		fieldContributor.contribute( absoluteFieldPath, value, documentBuilder::addField );
+		List<IndexableField> fields = new ArrayList<>();
+		fieldContributor.contribute( absoluteFieldPath, value, fields::add );
```

Comment:
You could just use a lambda that does exactly what the body of your "for" loop does. Metaspace consumption problems are the same with lambdas and method references anyway, and you would at least avoid instantiating an `ArrayList` each time the user adds a field (especially since, in this case, the list will be instantiated even if the user doesn't add anything...)

Reply:
Yup, it should have been a lambda. Forgot to commit my changes on Friday evening...

Changed and merged, thanks for the review.